scripts/genAbi.js

```diff
@@ -32,16 +33,17 @@ async function main() {
     cwd: nitroPath,
   })
 
-  console.log('building peripherals')
-  execSync(`${npmExec} && ${npmExec} run hardhat compile`, {
-    cwd: peripheralsPath,
```

Comment:
I don't think the peripheral build should be removed

Reply:
whenever i try to `hardhat compile` in that directory it just says nothing to compile since there's no hardhat config. the npm package also already has the `build` dir

Reply:
I see, #346 bumped from classic peripheral to the new token-bridge-contracts repo. That would require some additional hacking since the hardhat config is removed from the token-bridge-contracts release. An example fix is here https://github.com/OffchainLabs/arbitrum-sdk/pull/295

Reply:
ok cool, I'll do something like that in 346 then